src/components/JsonSchemaValidator/Schemas/dbtAttribution_0.2.0.json

```diff
@@ -120,7 +120,7 @@
       "description": "> Click the plus sign to add a new entry",
       "minItems": 0,
       "title": "Path Transforms",
-      "longDescription": "Dictionary of path transforms (and their argument, `null` if none) to perform on the full conversion path (see the transform path options in our [docs](/docs/modeling-your-data/modeling-your-data-with-dbt/package-mechanics/manifest-tables/)).",
+      "longDescription": "Dictionary of path transforms (and their argument, `null` if none) to perform on the full conversion path (see the transform path options in our [model docs](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/)).",
```

Comment:
Aren't we supposed to add the `index.md` at the end to let docusaurus automatically check for broken links?

Reply:
Normally yes, but these get rendered in a MUI table as direct links so the index.md breaks it 
![image](https://github.com/snowplow/documentation/assets/8260415/64ed5cbd-dea5-4c8e-a9b5-2a8c9a2c0f3f)